plugins/modules/rds_instance_snapshot.py

```diff
@@ -8,11 +8,11 @@
 __metaclass__ = type
 
 
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 ---
 module: rds_instance_snapshot
 version_added: 1.0.0
-short_description: manage Amazon RDS snapshots.
+short_description: Manage Amazon RDS instance snapshots.
```

Comment:
```suggestion
short_description: Manage Amazon RDS instance snapshots
```
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-block
`Unlike description:, short_description should not have a trailing period/full stop.`